cms/test_utils/project/placeholderapp/views.py

```diff
@@ -24,7 +25,12 @@ def _base_detail(request, instance, template_name='detail.html',
         request.toolbar.set_object(instance)
     if template_string:
         template = Template(template_string)
-        return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        if DJANGO_1_7:
+            return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        else:
+            from django.template.context import make_context
+            context = make_context(context, request)
+            return HttpResponse(template.render(context))
```

Comment:
What deprecation does this fix? I guess I'm skeptical of the use of `make_context` since that's not a public API.